### PR TITLE
Update Owner help content for Transfers and Registrations

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#19492

*Description of changes:*
- Update Owner help content for Transfers and Registrations

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1777-31x8zj41.web.app

File: ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import ErrorContact from '@/components/common/ErrorContact.vue'
+
+export default {
+  components: {
+    ErrorContact
+  }
+}
+</script>
+
+
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue]: Maybe `script setup`?

File: ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+h4 {
+  font-size: 16px;
+  line-height: normal
+}
+
+p {
+  line-height: 24px;
+  margin-bottom: 24px;
+  color: gray7;
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue]: I believe $gray7 here? That being said, i think it's $gray7 by default?

**Review by cameron-eyds — APPROVED** [review]
LG!

File: ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue]: Good catch, i removed default styles.

File: ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/HelpWithOwners.vue]: Cleaned up to use script setup

**Comment by dimak1** [comment]
gcbrun

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1777-31x8zj41.web.app

**Review by doug-lovett — APPROVED** [review]
Nice.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1777-31x8zj41.web.app

**Pull request merged**